docker/libstuff/bedrock.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -e
+
+# Requires bash 4.2 or greater because of use of `-v` for testing variables being set
+# Returns 1 if version doesn't match
+function bashVersionCheck()
+{
+  [ -z $BASH_VERSIO ] && return 1
+
+  # If it's set, check the version
+  case $BASH_VERSIO in
+    5.*) return 0 ;;
+    4.1) return 1 ;;
+    4.0) return 1 ;;
+    4.*) return 0 ;;
+    ?) return 1;;
+  esac
+}
+
+if [ $(bashVersionCheck) ]; then
```

Comment:
We prefer to use `[[` over `[`

```suggestion
if [[ $(bashVersionCheck) ]]; then
```